src/js/shot-generator/apply-device-quaternion.js

```diff
@@ -4,7 +4,8 @@ const applyDeviceQuaternion = ({
   startingObjectOffset,
   startingObjectQuaternion,
   deviceQuaternion,
-  camera
+  camera,
+  isXRController = false
```

Comment:
Could we use a more descriptive name for the argument? Something like: `useCameraOffset`?